aip/general/0133.md

```diff
@@ -126,8 +126,18 @@ to done if the request is effectively immediate.
 
 ### User-specified IDs
 
-An API **must** allow a client to specify the ID component
-of a resource (the last segment of the resource name) on creation. An API
+An API **must** allow a user to specify the ID component of a resource (the last
```

Comment:
Any more updates planned to this section, such as the global namespace issue, whether renaming is needed, whether the resource is created as a side effect of a custom method rather than the standard Create method, etc.?

Reply:
Yes